apps/omg_api/lib/state/core.ex

```diff
@@ -428,6 +435,67 @@ defmodule OMG.API.State.Core do
     {:ok, {event_triggers, db_updates, validities}, new_state}
   end
 
+  defp sigs_chope(<<>>), do: []
```

Comment:
spelling + wouldn't this fit better in `Transaction.Signed`? you even have a `@signature_length` defined there